### ISSUE TITLE
Customer Vs Vendor Order Details

### ISSUE THREAD

**Issue by prabhupdesh** [BPS-Industries/Navision#56]
Buyer PO Details | Raw Material Details (UBOM) | Vendor Purchase Order Details
-- | -- | --
Nav SO no. | Customer Season | Buyer No. | Buyer Name | Customer PO | Nav FG No. | Nav FG Description | Style No. | Style Name | Shipped Status | Prod Order Status (Mat. issued to line) | Quantity | Price | Order Date | Shipper (Entity) | Shipping from | Shipment Mode | Requested Ship Date | RM Product Group | RM Item No. | RM Item Name | RM Quantity per | RM UOM | Vendor Purchase Order No. | Vendor PO Line No. (Nav) | PO Vendor Name | Promised Delivery Date | Quantity Ordered | Item No,. | Quantity Received | Pending Quantity to Receive | Total Value (LCY) | Cancellable/Deferrable

**Comment by prabhupdesh** [comment]
LE,PU,LI,HW,PK-TI,PK-BX,PK-HE,PK-HA,PK-ID,PK-BB,OV-OT (with PU laminate in description) should only be  shown for purchase order

**Comment by prabhupdesh** [comment]
Create a table "Vendor PO Upload Set Nos."

Fields:

Purchase Order No. (PK)
Purchase Order Line No. (PK)
Upload Set No. (PK)
Created By
Creation date
Last Modified By
Last Modification Date

**Comment by YashwanthGowdaJ** [comment]
Created a page for vendor PO upload set Nos, and included that page in purchase order and also in purchase order subform. They can enter data only through subform, we have made vendor PO upload set No page as non editable from purchase order. 

Added a new field in Vendor PO upload set Nos called as link type with 2 options, PO & VDF. By default it will be PO. If link type is PO, then upload set numb lookup will be for customer so buffer or else for production forecast
